### ISSUE TITLE
change index.js point to './src/core' break test cases follow README

### ISSUE THREAD

**Issue by dead-horse** [facebook/jscodeshift#206]
this commit https://github.com/facebook/jscodeshift/commit/964f0a9c9182fe2172efc5c6c3ef6097ffe00e27 change index.js to `module.exports = require('./src/core')`.

 but https://github.com/facebook/jscodeshift#unit-testing use `const defineTest = require('jscodeshift/dist/testUtils').defineTest;`, that makes `jscodeshift.registerMethods` useless in our test case.

**Comment by fkling** [comment]
You can use `require('jscodeshift/src/testUtils')` instead. But I agree that this `dist/` / `src/` situation is not ideal and confusing. Will have to think about how to solve that.

**Comment by dead-horse** [comment]
yes, I can use `require('jscodeshift/src/testUtils')`, but it is a breaking change for user because it is recommend in README.
